tests/foreman/cli/test_satellitesync.py

```diff
@@ -2022,6 +2010,97 @@ def test_positive_export_import_incremental_yum_repo(
         export_cc['packages'] = str(int(export_cc['packages']) + 1)
         assert import_repo['content-counts'] == export_cc, 'Import counts do not match the export.'
 
+    @pytest.mark.tier3
+    @pytest.mark.parametrize(
+        'function_synced_rhel_repo',
+        ['rhae2'],
+        indirect=True,
+    )
+    def test_positive_export_import_mismatch_label(
+        self,
+        target_sat,
+        export_import_cleanup_function,
+        config_export_import_settings,
+        function_sca_manifest_org,
+        function_import_org_with_manifest,
+        function_synced_rhel_repo,
+    ):
+        """Export and import repo with mismatched label
+
+        :id: eb2f3e8e-3ee6-4713-80ab-3811a098e079
+
+        :setup:
+            1. Enabled and synced RH yum repository.
+
+        :steps:
+            1. Export and import a RH yum repo, verify it was imported.
+            2. Export the repo again and change the repository label.
+            3. Import the changed repository again, it should succeed without errors.
+
+        :expectedresults:
+            1. All exports and imports succeed.
+
+        :CaseLevel: System
+
+        :CaseImportance: Medium
+
+        :BZ: 2092039
+
+        :customerscenario: true
+        """
+        # Verify export directory is empty
+        assert target_sat.validate_pulp_filepath(function_sca_manifest_org, PULP_EXPORT_DIR) == ''
+        # Export the repository and check the export directory
+        export = target_sat.cli.ContentExport.completeRepository(
+            {'id': function_synced_rhel_repo['id']}
+        )
+        assert '1.0' in target_sat.validate_pulp_filepath(
+            function_sca_manifest_org, PULP_EXPORT_DIR
+        )
+
+        # Run import and verify the product and repo is created in the importing org
+        import_path = target_sat.move_pulp_archive(function_sca_manifest_org, export['message'])
+        target_sat.cli.ContentImport.repository(
+            {'organization-id': function_import_org_with_manifest.id, 'path': import_path},
+            timeout=300000,
```

Comment:
Is this a 300000 second timeout? If so, I think we should turn it down.

Reply:
Should be in miliseconds I guess (at least from what I can see elsewhere grepping for `\.cli\..*timeout`)

Reply:
I followed the steps to here https://github.com/SatelliteQE/broker/blob/master/broker/hosts.py#L139. It's possible VS Code led me astray, so I'll double check.

Reply:
Might be good to tag @JacobCallahan here for confirmation.

Reply:
@Griffin-Sullivan is correct, that timeout is in seconds. In fact, Broker also allows you to use more clear timeout formatting so instead of specifying a long timeout like that, you can just pass `timeout="5m"`
https://github.com/SatelliteQE/broker/blob/master/broker/session.py#L99
https://github.com/SatelliteQE/broker/blob/master/broker/helpers.py#L417

Reply:
Ok, changed to the nicer format, let's wait for the PRT.